CONTRIBUTING.md

```diff
@@ -53,9 +53,19 @@ can continue to contribute to the project using the standard `npm` workflow, ado
 
 ## Setting up your environment
 
-Want to get started hacking on the code? Super! Follow these instructions to get up and running.
+Want to get started hacking on the code? Super!
 
-First, make sure you have the prerequisites installed and available on your `$PATH`:
+### Using Visual Studio Code
+
+We love [Visual Studio Code](https://code.visualstudio.com/) for many reasons, mainly:
+
+- You can debug JavaScript/TypeScript code right away with [automatic debugging configuration](https://code.visualstudio.com/updates/v1_45#_automatic-debug-configurations).
+- You can use it with Github's [Codespaces](https://visualstudio.microsoft.com/services/github-codespaces/) to develop inside a docker container that has all the prerequisites.
+- You get [excellent support for TypeScript](https://code.visualstudio.com/Docs/languages/typescript).
+
+### Prerequisites
+
+If you do not use github's Codespaces, make sure you have the prerequisites installed and available on your `$PATH`:
```

Comment:
> If you do not use...

This would be a valid statement to make if we have already spoken about how to use codespaces with this repo. Are there any instructions we can add for that? Then we can update the above to be

```suggestion
If you are not using GitHub's Codespaces, make sure you have the prerequisites installed and available on your `$PATH`:
```

Reply:
I pushed this change: https://github.com/Azure/azure-sdk-for-js/pull/11920/commits/e8de80e5cc847fa48ba0b5004b250883c821153e.